fem/lininteg.hpp

```diff
@@ -27,6 +27,15 @@ class LinearFormIntegrator
    LinearFormIntegrator(const IntegrationRule *ir = NULL) { IntRule = ir; }
 
 public:
+
+   /// Method probing for assembly on device
+   virtual bool UseDevice() { return false; }
```

Comment:
I think `SupportsDevice` would be clearer

Reply:
`SupportsDevice`